lib/Client/LogRecord.php

```diff
@@ -10,13 +13,15 @@
 class LogRecord
 {
     protected $_fields = null;
```

Comment:
nit: let's either decide to use `null` or `NULL` but not both

Reply:
done